pytim/gitim.py

```diff
@@ -199,24 +202,19 @@ def alpha_shape(self, alpha, group, layer):
             extraids = np.append(extraids, [-1] * n_cube)
         else:
             n_cube = 0
-            extrapoints = np.copy(points)
+            extrapoints = points
             extraids = np.arange(len(points), dtype=np.int)
 
-        self.triangulation.append(Delaunay(extrapoints))
-        try:
```

Comment:
Please keep this try-except block. The user can still call alpha_shape independently, and passing the wrong layer id.

Reply:
Ok, but why can a user call alpha_shapes independently?

Reply:
It's handy to have a method to compute the alpha shapes. And a try/except does not cost anything but can save you a lot of time finding out what went wrong.